src/javascript/typescript/language.ts

```diff
@@ -29,7 +28,7 @@ export interface TypescriptCompilerOptions extends CompilerOptions {
 
 export default class TypescriptGenerator {
   options: TypescriptCompilerOptions
-  typeFromGraphQLType: (graphQLType: GraphQLType, options?: TypeFromGraphQLTypeOptions) => t.TSType
+  typeFromGraphQLType: Function
```

Comment:
Do we want to type this with a stricter signature? It will help catch errors if we refactor it in the future.

Reply:
Good point. Let's do that